test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -27,23 +28,44 @@
 MNIST_PATH = os.path.join(RESOURCE_PATH, 'mnist')
 THROUGHPUT_PATH = os.path.join(RESOURCE_PATH, 'smdataparallel')
 
+
+def validate_or_skip_smdataparallel(ecr_image):
+    if not can_run_smdataparallel(ecr_image):
+        pytest.skip("Data Parallelism is supported on CUDA 11 on TensorFlow 2.3.1 and above")
+
+
+def can_run_smdataparallel(ecr_image):
+    _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+    image_cuda_version = get_cuda_version_from_tag(ecr_image)
+    return Version(image_framework_version) in SpecifierSet(">=2.3.1") and Version(
+        image_cuda_version.strip("cu")) >= Version("110")
```

Comment:
nit: You can just compare two versions with `Version(x) >= Version(y)`, but using `SpecifierSet`s is also good.